packages/changeset-validations/package.json

```diff
@@ -31,7 +31,8 @@
   "peerDependencies": {
     "validated-changeset": "^1.3.4",
     "ember-changeset": "^4.1.2",
-    "ember-headless-form": "^0.0.0"
+    "ember-headless-form": "^0.0.0",
+    "ember-source": "^4.8.0"
```

Comment:
Is the min version 4.4 or 4.8?

The try config has 4.4. last i looked, but if 4.8 is min required, then we can remove the function polyfill 🎉